### PR TITLE
Update Command class and functions in Command

### PR DESCRIPTION
**Pull request by hlwang56**
//may merge it later cuz there's still some bugs in there that need to be fixed

### CODE REVIEW

File: src/main/java/commands/Command.java [hunk 1]
```diff
@@ -13,48 +16,118 @@
...
+        ui.printMsg(replyMsg);
     }
-    */
+
+    private void deleteSpend() {
+        Person newPerson;
+        String uid;
+        int index;
+        uid = parser.parseUserIndex(userInput);
+        index = parser.parseRecIndex(userInput);
```
Comment (Teanweijun) [inline src/main/java/commands/Command.java:79]: I think it'll be cleaner if you can initialise these 3 variables in the same line which you declared them.
Similar for functions below

File: src/main/java/commands/Command.java [hunk 2]
```diff
@@ -13,48 +16,118 @@
...
+        Person newPerson;
+        String description;
+        description = parser.parseDescription(userInput);
+        amount = parser.parseIncome(userInput);
+        uid = parser.parseUserIndex(userInput);
+        newPerson = personList.getPerson(uid);
+        newPerson.addIncome(description, amount);
+    }
+
+    private void deletePerson() throws Exception{
```
Comment (Teanweijun) [inline src/main/java/commands/Command.java:118]: Small syntax issue, missing space between `Exception` and `{`

File: src/main/java/seedu/planitarium/PlanITarium.java [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
+            commandExecuter = new Command(userInput.nextLine());
+            commandExecuter.execute();
+        }
     }
 
     /**
      * Entry-point for the PlanITarium application.
      */
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception{
```
Comment (Teanweijun) [inline src/main/java/seedu/planitarium/PlanITarium.java:24]: Same syntax issue as above

**Review by Teanweijun — COMMENTED** [review]
Overall looks good, but can be improved by removing redundant lines to make the code neater

**Review by 1szheng — APPROVED** [review]
Changes made resolves comments. Good job.

**Pull request merged**